src/i18n/strings/en_EN.json

```diff
@@ -197,13 +201,14 @@
   "Claimed Ed25519 fingerprint key": "Claimed Ed25519 fingerprint key",
   "Clear Cache and Reload": "Clear Cache and Reload",
   "Clear Cache": "Clear Cache",
-  "Click here": "Click here",
```

Comment:
For all cases where I have removed strings, I have manually grepped both react-sdk/riot-web to make sure it was only used in the 1 place and was safe to remove.